graphql/executor.go

```diff
@@ -366,6 +366,9 @@ func (e *Executor) execute(ctx context.Context, typ Type, source interface{}, se
 	}
 	switch typ := typ.(type) {
 	case *Scalar:
+		if typ.Unwrapper != nil {
```

Comment:
I think this commit should be further up the chain. Looks like the dates might be messed up on the commits?

Reply:
it's the correct order in the stack, but github is stupid.